Tests/Parser/ABOParserTest.php

```diff
@@ -2,9 +2,11 @@
 
 namespace JakubZapletal\Component\BankStatement\Tests\Parser;
 
+use JakubZapletal\Component\BankStatement\Tests\TestCase;
 use JakubZapletal\Component\BankStatement\Parser\ABOParser;
 
-class ABOParserTest extends \PHPUnit_Framework_TestCase
+
```

Comment:
Is there any reason why do you keep 2 blank lines between `use` and `class`? It's in all classes.